package.json

```diff
@@ -83,6 +83,8 @@
     "webpack": "^3.6.0"
   },
   "scripts": {
+    "lint": "yarn lint-scripts && yarn lint-styles && yarn lint-php",
+    "lint-php": "~/.composer/vendor/bin/phpcs .",
```

Comment:
I think this should be changed to just `phpcs .` as there's a number of factors preventing it from working; custom composer directories and that only *nix based systems use a `~/.composer` directory. It won't work on Windows; https://getcomposer.org/doc/03-cli.md#composer-home

Reply:
Travis doesn't export composer binaries to $PATH, so we have to quantify it.
Thankfully, there's a command for that.

Reply:
Perfect 👍